src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -101,6 +101,22 @@ public function getAddress()
         return $this->hostname.':'.$this->port;
     }
 
+    /**
+     * @return string contains resolved hostname if available, empty string otherwise
+     */
+    public function getExtraAddress()
```

Comment:
What about renaming this method as `getRealAddress()` or `getResolvedAddress()` ?